ee-9/feature-pack/pom.xml

```diff
@@ -817,6 +823,10 @@
                     <groupId>${project.groupId}</groupId>
                     <artifactId>jipijapa-eclipselink</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>${ee.maven.groupId}</groupId>
+                    <artifactId>jipijapa-hibernate5-3</artifactId>
```

Comment:
:+1: for excluding the jipijapa-hibernate5-3, soon that will be removed but not yet.